effekt/shared/src/main/scala/effekt/generator/llvm/Transformer.scala

```diff
@@ -26,9 +25,20 @@ object Transformer {
 
       val entryBlock = BasicBlock("entry", instructions, terminator)
       val entryFunction = Function(VoidType(), "effektMain", List(), entryBlock :: basicBlocks)
-      declarations.map(transform) ++ definitions :+ entryFunction
+      declarations.map(transform) ++ definitions ++ staticTextDefinitions(MC.staticText) :+ entryFunction
   }
 
+  // context getters
+  private def MC(using MC: ModuleContext): ModuleContext = MC
+  private def FC(using FC: FunctionContext): FunctionContext = FC
+  private def BC(using BC: BlockContext): BlockContext = BC
+
+  private def staticTextDefinitions(staticText: Map[String, Array[Byte]]): List[Definition] =
+      staticText.map { (global, bytes) =>
+        val escaped = bytes.map(b => f"\$b%02x").mkString;
```

Comment:
Could you explain this to me?

Reply:
For each statically known byte arrays (currently these are utf-8-encoded string literals) `staticTextDefinitions` builds a global LLVM-definition of the form `@litral.932 = private constant [4 x i8] c"\x75\x74\x66\x38"`.

Reply:
The above is now still true, only `staticText` vanished: [2366c19](https://github.com/effekt-lang/effekt/pull/142/commits/2366c19404fed6660103f4cb166567cecbbfdb52)